invenio_rdm_records/fixtures/data/vocabularies/resource_types.yaml

```
@@ -3,8 +3,8 @@
   props:
     csl: report
     datacite_general: Text
-    datacite_type: ''
-    openaire_resourceType: '17'
+    datacite_type: Text
```

```suggestion
    datacite_type: ''
```

If the general type doesn't differ from the `datacite_type`, we can keep the string empty.

---

Same applies to cases where the string is two words (e.g. `Conference paper` is not needed, since the general type is `ConferencePaper`)

---

Changes applied. I'm unsure about putting an empty string for those that are two words since the general one removes the space.

Would it be worth documenting how to update these vocabularies? It was not trivial to understand what each field is used for.